### PR TITLE
[DEBT] Unified place to get expected marks

### PR DESCRIPTION
**Pull request by dmitry-uraev**

### CODE REVIEW

File: config/get_mark.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+set -ex
+
+if [[ $1 == "crawler" ]]; then
+  export TARGET_SCORE=$(head -5 config/target_score.txt | tail -1)
```
Comment (demid5111) [inline config/get_mark.sh:4]: this is wrong! here you get the mark for pipeline(!!)

File: config/get_mark.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+elif [[ $1 == "pipeline" ]]; then
+  export TARGET_SCORE=$(head -2 config/target_score.txt | tail -1)
```
Comment (demid5111) [inline config/get_mark.sh:6]: the same (!)

**Review by demid5111 — CHANGES_REQUESTED** [review]

File: config/get_mark.sh (hunk 1 above)
Comment (dmitry-uraev) [reply config/get_mark.sh]: fixed

File: config/get_mark.sh (hunk 2 above)
Comment (dmitry-uraev) [reply config/get_mark.sh:4]: fixed

**Pull request merged**